sdk/src/feature_set.rs

```diff
@@ -1016,6 +1020,7 @@ lazy_static! {
         (migrate_feature_gate_program_to_core_bpf::id(), "Migrate Feature Gate program to Core BPF (programify) #1003"),
         (vote_only_full_fec_sets::id(), "vote only full fec sets"),
         (migrate_config_program_to_core_bpf::id(), "Migrate Config program to Core BPF #1378"),
+        (zk_elgamal_proof_program_enabled::id(), "Enable ZkElGamalProof program"),
```

Comment:
Can you add the feature gate issue number here?

Reply:
Okay, I think I will need to create a SIMD for this first, so I will come back to this after.

Reply:
Just to be clear, we can merge this as is and add the SIMD later

Reply:
Sorry, since there was a conflict with the master branch, I could not merge right away and had to rebase. Now that SIMD is up, I just added the SIMD number in the description while I was rebasing.